package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "sdc-clients",
   "description": "Contains node.js client libraries for SDC REST APIs.",
-  "version": "9.2.0",
+  "version": "9.3.0",
   "homepage": "http://www.joyent.com",
   "repository": {
```

Comment:
Should update assert-plus to `1.*` latest, it has fixes for a number of methods, esp. the "optional*" ones.